### ISSUE TITLE
`Search Gnus(imap)` option get `gnus-get-function: No such function: nnir-server-opened` error

### ISSUE THREAD

**Issue by hwiorn** [deusmax/gnus-notes#11]
In some org file, `M-x gnus-notes-org-handle-mail` and I pressed `v (Search Gnus)`.
But I got the `gnus-get-function: No such function: nnir-server-opened` error.

I think this issue is related with `nnir` deprecated now.

How can I use this option?

- Emacs: 28.2
- Gnus: v5.13

**Comment by deusmax** [comment]
Correct. It does a `nnir` search on the server for any gnus-links in the text body of the current heading.

As `nnir` is deprecated, I will provide a fix. Time to catch-up on the new gnus search method.
Thanks.
